controllers/api.controller.js

```diff
@@ -15,3 +16,11 @@ exports.getTopics = (req, res, next) => {
     .catch((err) => {
```

Comment:
What is this catch doing? There is no use of next here, nor are there any real reasons to catch on this endpoint